packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java

```diff
@@ -434,7 +426,7 @@ private void applySettings(Map<String, Object> settings) {
     }
   }
 
-  private void updateJsMode(int mode) {
+  private void updateJsMode(WebView webView, int mode) {
```

Comment:
This method can be marked as `static`. This would have a small performance advantage and also guards against usage of instance members.